packages/swapper/src/swappers/thorchain/utils/makeSwapMemo/makeSwapMemo.ts

```diff
@@ -19,7 +21,8 @@ export const makeSwapMemo = ({
   destinationAddress: string
   limit: string
 }): string => {
-  const thorId = adapters.assetIdToPoolAssetId({ assetId: buyAssetId })
+  const isRune = buyAssetId === thorchainAssetId
```

Comment:
can we use our little function above here?

https://github.com/shapeshift/lib/pull/1030/files#diff-beaf016013438f20332d1ae30ebda43adf09fa660169bfb5c1edde2d3ff36f4fR3